requirements.txt

```diff
@@ -23,6 +23,7 @@ eth-utils==1.2.0
 structlog
 colorama
 py-solc
+pytest
```

Comment:
Why `pytest` in `requirements.txt`?

Reply:
smoketest uses it :(

Reply:
Ok, need to remove it from `requirements-dev.txt` then, it breaks being in both

Reply:
what breaks?

Reply:
https://travis-ci.org/raiden-network/raiden/jobs/428128989#L579

Reply:
ohh, gracias